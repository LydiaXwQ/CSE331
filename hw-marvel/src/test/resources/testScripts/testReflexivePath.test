# UWProfessorsCustom.csv is a file made by Rich Chen. This csv file includes UW CSE professors and
# the courses they've taught/are teaching/will be teaching from 2021-2023.

# Test finding the path from a node to itself

LoadGraph prof UWProfessorsCustom.csv

FindPath prof Ru.Anderson Ru.Anderson