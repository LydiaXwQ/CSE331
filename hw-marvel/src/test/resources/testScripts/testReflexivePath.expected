# UWProfessorsCustom.csv is a file made by Rich Chen. This csv file includes UW CSE professors and
# the courses they've taught/are teaching/will be teaching from 2021-2023.

# Test finding the path from a node to itself

loaded graph prof

path from Ru.Anderson to Ru.Anderson: